net-analyzer/prelude-lml/prelude-lml-5.1.0.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools systemd
+
+DESCRIPTION="The prelude log analyzer"
+HOMEPAGE="https://www.prelude-siem.org"
+SRC_URI="https://www.prelude-siem.org/pkg/src/${PV}/${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="icu ssl"
+
+RDEPEND="dev-libs/libpcre
+    >=dev-libs/libprelude-5.1.0
+	icu? ( dev-libs/icu:= )
+	ssl? ( net-libs/gnutls:= )"
+
+BDEPEND="${RDEPEND}
```

Comment:
I've explained the roles of `RDEPEND`, `DEPEND` and `BDEPEND` in a previous comment. Please fix this accordingly. Let me know if there are any confusions.

Reply:
done